examples/text_classification/iterable_train.py

```diff
@@ -176,7 +175,7 @@ class Dataset(torch.utils.data.IterableDataset):
         num_lines: the number of lines read by the individual iterator.
     """
     def __init__(self, iterator, num_lines):
-        super(Dataset, self).__init__()
+        super().__init__()
```

Comment:
Is this fix relevant to the drop of python 2 support? If not, I suggest to fix in a separate issue.

Reply:
Reverted. Please could you add a comment on which change you're referring to below?